src/main/java/org/wildfly/security/x500/X500.java

```diff
@@ -18,8 +18,14 @@
 
 package org.wildfly.security.x500;
 
+import org.wildfly.security._private.ElytronMessages;
+
 import java.security.cert.X509Certificate;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Properties;
```

Comment:
I think this new functionality may make more sense as part of the general ASN1 class as OIDs go beyond X.500.